### PR TITLE
Updated Readm.md

### PR DESCRIPTION
**Pull request by slim-python**
Provided solution to Fix "whynotwin11.exe is not commonly downloaded and may be dangerous"

### CODE REVIEW

**Review by rcmaehl — CHANGES_REQUESTED** [review]
Formatting needs to be fixed a bit so the images don't have numbers, and some text cleanup

File: README.md [hunk 1]
```diff
@@ -18,7 +18,26 @@ Detection Script to help identify why your PC isn't Windows 11 ready
...
+4. ![enter image description here](https://windowsground.com/wp-content/uploads/2021/06/download-whynotwin11-tool.png)
+5. If it asks for confirmation, click on keep anyway
+6. ![enter image description here](https://windowsground.com/wp-content/uploads/2021/06/install-whynotwin11-tool.png)
+7. Let it download fully and then install it
+8. when you get a popup like this, click on **More info**
+9. ![enter image description here](https://lh4.googleusercontent.com/d2lfHx2KYAfm783tVZvgkGKjEpRy2b2Dy15mlgJ5RNU69o4-3I2Mw-T54I7WIMWQHrRixSDS0qlA-Y6WhUArJOfJL6CnFvUMtgcaC5nj5NQLQhm2gXWJsoMeZZFv3FGfkk9ekItj)
+10. Next click on Run anyway
+11. ![enter image description here](https://windowsground.com/wp-content/uploads/2021/06/run-anyway-whynotwin11-tool.png)
+
+If you want a detail guide you can check [here](https://windowsground.com/download-whynotwin11-compatibility-checker-tool-to-check-if-you-can-upgrade-to-windows-11/)
```
Comment (micwoj92) [inline README.md:39]: What does it mean? the guide you posted here is the same to the one on the site.

Comment (slim-python) [reply README.md:39]: yes, but a little bit detailed, thought it would be helpful

Comment (micwoj92) [reply README.md:39]: The guide texts is literally 1:1 same

**Comment by slim-python** [comment]
> Formatting needs to be fixed a bit so the images don't have numbers and some text cleanup

um okay, you decide, I have updated the readme from my side

**Comment by billysielu** [comment]
Looks like this relates to downloading dev builds. Maybe move the dev builds link way further down the page as that's not the one most people want.

**Comment by slim-python** [comment]
> Looks like this relates to downloading dev builds. Maybe move the dev builds link way further down the page as that's not the one most people want.
> 
> Edit: Also, the web link doesn't explain what yellow ? means, which is probably a helpful thing it could do.

it does, check at the end

File: README.md [hunk 2]
```diff
@@ -18,7 +18,33 @@ Detection Script to help identify why your PC isn't Windows 11 ready
...
+## Fix whynotwin11.exe is not commonly downloaded and may be dangerous
+![## Fix whynotwin11.exe is not commonly downloaded and may be dangerous](https://i.imgur.com/sVvwCLz.png)
+
+If your computer is blocking you to download this and showing you the above error, follow these steps to download easily:
+
+1. Download the whynotwin11tool from [here](https://github.com/rcmaehl/WhyNotWin11/releases/latest/download/WhyNotWin11.exe)
+2. Go to your downloads tab by pressing **ctrl + j**
+3. Expand and click on **keep** 
+
+![enter image description here](https://windowsground.com/wp-content/uploads/2021/06/download-whynotwin11-tool.png)
```
Comment (schmelto) [inline README.md:31]: Consider using image descriptions.

**Comment by micwoj92** [comment]
To be honest I think that it should be common knowledge on how to disable
such warnings. I think this pr should be closed.

W dniu wtorek, 29 czerwca 2021 Tom Schmelzer ***@***.***>
napisał(a):
> @schmelto commented on this pull request.
>
> ________________________________
>
> In README.md:
>
>> @@ -18,7 +18,33 @@ Detection Script to help identify why your PC isn't
Windows 11 ready
> [Download here](
https://github.com/rcmaehl/WhyNotWin11/releases/latest/download/WhyNotWin11.exe
)
>
> [Download DEV BUILDS here](
https://github.com/rcmaehl/WhyNotWin11/actions?query=workflow%3AWNW11)\
> -/!\ Dev builds will not be able to automatically check for updates
> +/!\ Dev builds will not be able to automatically check for updates
> +## Fix whynotwin11.exe is not commonly downloaded and may be dangerous
> +## Fix whynotwin11.exe is not commonly downloaded and may be
dangerous
> +
> +If your computer is blocking you to download this and showing you the
above error, follow these steps to download easily:
> +
> +1. Download the whynotwin11tool from [here](
https://github.com/rcmaehl/WhyNotWin11/releases/latest/download/WhyNotWin11.exe
)
> +2. Go to your downloads tab by pressing **ctrl + j**
> +3. Expand and click on **keep**
> +
> +enter image description here
>
> Consider using image descriptions.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or unsubscribe.<
https://ci6.googleusercontent.com/proxy/xgZUYv5IVLJfHTBN-DJda6WWN1h06ERQpkIXCw-vWcCU1whoIgf87Qd99b59y5Dx67YbPzDs-LXz7X3gD8fST1VZ9bIqH00OSMCjG838wxCL-TxRensDV_QvDGqi7Ed-qUj5tCZfnPTl4Waddn7p37c6ke3ZF5a1d146ibABYmXf0WOW6RXaPzMQRfv1TH47i92QX72gjRWC5Uc72JA1ofAsBwfWvh1VP5dHfJdRjPyRUCiJlGCZ=s0-d-e1-ft#https://github.com/notifications/beacon/AK3YG4UPJCMSQWEIF5SGZVLTVETRHA5CNFSM47MVNI52YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOFFS3CQA.gif>

**Pull request closed without merging**